pkg/jobs/jobs.go

```diff
@@ -113,16 +113,15 @@ func makeQueueName(domain, workerType string) string {
 
 func makeID() string {
 	const (
-		idLen         = 16
-		letters       = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
-		letterIdxMask = 1<<6 - 1 // 6 bits to represent a letter index
+		idLen   = 16
+		letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 	)
+	r := rand.New(rand.NewSource(time.Now().UTC().UnixNano()))
```

Comment:
I'd have created a new source for the whole jobs package, not one per id. But it should work too